test/nimble_template/addons/variants/phoenix/api/json_api_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule NimbleTemplate.Addons.Phoenix.Api.JsonApiTest do
+  use NimbleTemplate.AddonCase, async: false
+
+  describe "#apply/2" do
+    @describetag mock_latest_package_versions: [{:jsonapi, "1.3.0"}]
+
+    test "injects jsonapi to mix dependency", %{
+      project: project,
+      test_project_path: test_project_path
+    } do
+      in_test_project(test_project_path, fn ->
+        ApiAddons.JsonApi.apply(project)
+
+        assert_file("mix.exs", fn file ->
+          assert file =~ "{:jsonapi, \"~> 1.3.0\"}"
+        end)
+      end)
+    end
+
+    test "adds config for json_api in config/config.exs", %{
```

Comment:
```suggestion
    test "adds config for jsonapi in config/config.exs", %{
```

Reply:
Thanks @byhbt I have changed it in https://github.com/nimblehq/elixir-templates/pull/164/commits/795704734b550b36320749f0321e8ee64f0d8fc7